script.js

```diff
@@ -18,4 +18,222 @@ window.onclick = function global(event) {
 
 openBtn.addEventListener('click', openModal);
 closeBtn.addEventListener('click', closeModal);
-modalContainer.addEventListener('click', closeModal);
\ No newline at end of file
+modalContainer.addEventListener('click', closeModal);
+
+// Mobile Modal
+const seeProject = document.querySelectorAll('.seeProject');
+const mobileModal = document.querySelectorAll('.mobileModal');
+
+const description0 = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea";
+
+const projectObj = {
+  image: [
+    './assets/projectImage00.png',
+    './assets/projectImage01.png',
+    './assets/projectImage02.png',
+    './assets/projectImage03.png',
+    './assets/projectImage04.png',
+    './assets/projectImage05.png',
+  ],
+  imageDesktop: [
+    './assets/desktop-details00.png',
+    './assets/desktop-details01.png',
+    './assets/desktop-details02.png',
+    './assets/desktop-details03.png',
+    './assets/desktop-details04.png',
+    './assets/desktop-details05.png',
+  ],
+  name: [
+    'Keeping track of hundreds of components',
+    'Keeping track of hundreds of components',
+    'Keeping track of hundreds of components',
+    'Keeping track of hundreds of components',
+    'Keeping track of hundreds of components',
+    'Keeping track of hundreds of components',
+  ],
+  lang: ['Ruby on rails', 'css', 'JavScript'],
+  langDesktop: [
+    'Codekit',
+    'GitHub',
+    'JavaScript',
+    'Bootstrap',
+    'Terminal',
+    'Codepen',
+  ],
+  description: [
+    description0,
+    description0,
+    description0,
+    description0,
+    description0,
+    description0,
+  ],
+  liveLink: [
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+  ],
+  sourceLink: [
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+    'https://github.com/mavericks-db',
+  ],
+};
+
+function openMobileModal(e) {
+  const i = e.target.dataset.index;
+  mobileModal[i].style.display = 'block';
+  const div1 = document.createElement('div');
+  div1.classList.add('mobileMCContainer');
+
+  const img1 = document.createElement('img');
+  img1.classList.add('closeMobile');
+  img1.setAttribute('src', './assets/close-mobileModal.svg');
+  function closeMobileModal() {
+    mobileModal[i].style.display = 'none';
+    div1.remove();
+  }
```

Comment:
- [ ] Please make your popup window to be closed to the figma design some are not matching the Figma design yet: 
-  What if you add a fix position CSS property `position: fix;` ? That will help prevent your popup windows page to scroll. so the popup matches the Figma more closely (both views).
-  Don't forget adding the `z-index:` CSS property to your popup window page because as you can see on the mobile version, your page Overlapping the popup window and this is because of the z-index. Overlapping elements with a larger z-index cover those with a smaller one.




Your Implementation | Figma Design
:---:|:---:
![mobpop](https://user-images.githubusercontent.com/79058364/161746982-4d074851-3b6b-47c3-a8eb-e7a75b80b85d.png) | ![goodpopmobile](https://user-images.githubusercontent.com/79058364/161747369-ec03ea99-2765-44b2-801a-102ed003f51a.png)


[ ] Kindly make the under background blurry when the popup windows is open, this will make the popup matches the Figma more closely 👍🏿



Your Implementation | Figma Design
:---:|:---:
![deskpop](https://user-images.githubusercontent.com/79058364/161750467-8981b451-885c-41a6-a5c5-8c49a5b23fce.png)| ![dessssssssssssssssssssssssssss](https://user-images.githubusercontent.com/79058364/161750695-065dbf52-9b37-4ace-b217-510505df9617.png)